<?xml version="1.0" encoding="utf-8"?>
<Workflow WorkflowId="{9ad856d7-5691-44d9-b992-985e8ca0c127}" Name="LogException">
  <XamlFileName>/Workflows/LogException-9AD856D7-5691-44D9-B992-985E8CA0C127.xaml</XamlFileName>
  <Type>1</Type>
  <Subprocess>0</Subprocess>
  <Category>3</Category>
  <Mode>0</Mode>
  <Scope>4</Scope>
  <OnDemand>0</OnDemand>
  <TriggerOnCreate>0</TriggerOnCreate>
  <TriggerOnDelete>0</TriggerOnDelete>
  <AsyncAutodelete>0</AsyncAutodelete>
  <SyncWorkflowLogOnFailure>1</SyncWorkflowLogOnFailure>
  <StateCode>1</StateCode>
  <StatusCode>2</StatusCode>
  <CreateStage>40</CreateStage>
  <RunAs>1</RunAs>
  <SdkMessageId>{197202f7-b872-e811-8196-e0071b69c9d1}</SdkMessageId>
  <UniqueName>LogException</UniqueName>
  <IsTransacted>1</IsTransacted>
  <IntroducedVersion>10.11.2017</IntroducedVersion>
  <IsCustomizable>1</IsCustomizable>
  <FormId>{00000000-0000-0000-0000-000000000000}</FormId>
  <PrimaryEntity>none</PrimaryEntity>
  <LocalizedNames>
    <LocalizedName languagecode="1033" description="LogException" />
  </LocalizedNames>
</Workflow>